core/src/display_object/movie_clip.rs

```diff
@@ -315,6 +321,75 @@ impl<'gc> MovieClip<'gc> {
         MovieClipWeak(GcCell::downgrade(self.0))
     }
 
+    pub fn new_import_assets(
+        activation: &mut Avm2Activation<'_, 'gc>,
+        movie: Arc<SwfMovie>,
+        parent: Arc<SwfMovie>,
+    ) -> Self {
+        let num_frames = movie.num_frames();
+
+        let loader_info = if movie.is_action_script_3() {
+            // The root movie doesn't have a `Loader`
```

Comment:
This was copied from `fn player_root_movie`, right? I don't think the comment is relevant here.

But also, in general - is setting the `loader_info` actually observable here in any way? Could this work if the instantiation was much simpler, closer to `new_with_data`?

Reply:
I removed the loader info as it's not used